index.html

```diff
@@ -329,13 +365,32 @@
       <p>The <dfn id="dom-navigator-sendbeacon"><code>sendBeacon</code></dfn>
       method transmits data provided by the <a href=
       "#data-parameter"><code>data</code></a> parameter to the URL provided by
-      the <a href="#url-parameter"><code>url</code></a> parameter. User agents
-      MUST ignore any entity bodies returned in the response. User agents MAY
-      close the connection prematurely once they start receiving an entity
-      body. The User Agent SHOULD transmit data at the earliest available
-      opportunity, but MAY prioritize the transmission of data lower compared
-      to other network traffic. The User Agent SHOULD make a best effort
-      attempt to eventually transmit the data.</p>
+      the <a href="#url-parameter"><code>url</code></a> parameter, according to
+      the following rules:</p>
+      <ul>
+        <li>The user agent SHOULD restrict the maximum <code>data</code> size
+        to ensure that beacon requests are able to complete quickly and in a
+        timely manner.</li>
+        <li>The user agent SHOULD schedule transmission of provided data to
+        minimize resource (CPU and network) contention with other time-critical
+        and high priority work.</li>
+        <li>The user agent MAY delay transmission of provided data to optimize
+        network and energy efficiency - e.g. deliver immediately if the network
+        is active, or wait until network interface is active. However, the user
+        agent SHOULD NOT delay transmission indefinitely and ensure that
+        pending transmissions are periodically flushed even if there is no
+        other network activity.</li>
+        <li>The user agent MUST schedule immediate transmission of all beacon
+        requests when the document <code>visiblityState</code>
+        ([[!PAGE-VISIBILITY]]) transitions to <code>hidden</code>, and must
+        allow all such requests to run to completion without blocking other
+        time-critical and high-priority work.</li>
```

Comment:
@annevk in Chrome we treat these requests as "detached requests": they're allowed to proceed independent of the state of render and without blocking it in any way. Also, note that prefetch/prerender have same semantics: those requests need to be allowed to proceed between navigations to avoid unnecessary and duplicate fetches.